### DISCUSSION TITLE
LUA-Script (os.execute)

### DISCUSSION THREAD

**Discussion by natjaberlin** [jcorporation/mympd-scripts#10] [Q&A] [answered]
My simple script works at the command line, but not as a script call  [  _**os.execute("./new.sh")**_  ]  in mympd.
How do I search for the error/problem?

**Comment by jcorporation** [comment]
The log of myzshould report the error. In your case I think that you must specify the absolute path to the script.

**Reply by natjaberlin** [reply]
Thx, but there is no error:
**Script: Info Script folder executed successfully**

Working (bash):
_**mkdir /home/volumio/test2**_

Not working (Lua script):
**_os.execute("mkdir /home/volumio/test2")_**

Big question mark!

**Comment by jcorporation** [comment, marked as answer]
Sure that the mympd user has permissions to create a directory in this folder?

The myMPD debug log could show the exact problem.

**Reply by natjaberlin** [reply]
Good question. 
I added _**mympd**_ to **_sudo_** as a test, but no success.
Where can I find the logfile (loglevel=7) of mympd? I have already searched everywhere.

**Comment by jcorporation** [comment]
myMPD logs to stdout. If you run it it under systemctl, journalctl is your friend.

**Reply by natjaberlin** [reply]
Found, thanks!
It really seems to be a problem with the authorization:

**_Dez 07 22:29:37 media1 sudo[785]: pam_unix(sudo:auth): conversation failed
Dez 07 22:29:37 media1 sudo[785]: pam_unix(sudo:auth): auth could not identify password for [mympd]_**
